cluster-install/README.adoc

```diff
@@ -105,12 +107,12 @@ Create cluster with Route53 private hosted zone and VPC
       --zones us-east-1a,us-east-1b \
       --state s3://kubernetes-aws-config \
       --vpc $VPCID \
-      --network-cidr 10.1.0.0/16\
+      --network-cidr 10.1.0.0/16 \
       --ssh-public-key $mypubkey
 
 == Validate cluster
 
-    kops validate cluster
+    kops validate cluster --name cluster03.k8s-aws.internal
```

Comment:
ditto here

Reply:
`kops validate cluster` won't work with a --name. Either its specified in an export, or explicitly.

Reply:
You mean "without", not "with", right?

Reply:
sigh - yes, must be dust on my glasses :-)